lib/cocina/models/admin_policy.rb

```diff
@@ -11,10 +11,10 @@ class AdminPolicy < Struct
 
       # The version of Cocina with which this object conforms.
       # example: 1.2.3
-      attribute :cocinaVersion, Types::Strict::String.default(Cocina::Models::VERSION)
+      attribute :cocinaVersion, CocinaVersion.default(VERSION)
```

Comment:
Case in point: instead of saying `cocinaVersion` should be a string, it should be a `CocinaVersion`. Note too that the explicit `Cocina::Models::` scope here was unnecessary since it's the default scope in this context.

Reply:
reads more easily this way, for sure.